temoa/components/reserves.py

```
@@ -29,348 +44,447 @@
+                _r_g += '+' + r1r2
+        # Get all contributing valid processes in each period
+        for p in model.time_optimize:
+            valid_rtv = {
+                (r, t, v)
+                for r, t in gather_group_active_processes(model, _r_g, p, t_g)
+                for v in model.process_vintages.get((r, p, t), set())
+            }
+            if valid_rtv:
+                model.operating_reserve_processes[(r_g, p, t_g)] = valid_rtv
+            else:
+                logger.info(
```

Ruff (0.16.1)

[error] Not enough arguments for `logging` format string

(PLE1206)